manta-js/examples/asset-webpack-ts/index.ts

```diff
@@ -62,6 +63,24 @@ const publicTransferTest = async () => {
     console.log("Dest Balance After:" + JSON.stringify(destinationBalanceAfterTransfer.toString()));
 }
 
+const publicTransferOnlySignTest = async () => {
+
+    const privateWalletConfig = {
+        environment: Environment.Production,
+        network: Network.Dolphin
+    }
+
+    const privateWallet = await MantaPrivateWallet.init(privateWalletConfig);
+
+    const destinationAddress = "5FHT5Rt1oeqAytX5KSn4ZZQdqN8oEa5Y81LZ5jadpk41bdoM";
+    const assetId = new BN("1"); // DOL
+    const amount = new BN("10000000000000000000"); // 10 units
```

Comment:
18 decimal points here instead of 12

Reply:
Yeah nice catch, the rest are fixed in the other PR.